unit_tests/datastore/test__api.py

```diff
@@ -17,6 +17,21 @@
 from google.cloud.datastore._api import HAVE_GRPC
 
 
+class TestDatastoreAPIBase(unittest.TestCase):
+
+    def _getTargetClass(self):
+        from google.cloud.datastore._api import DatastoreAPIBase
+        return DatastoreAPIBase
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def test__lookup_virtual(self):
+        api_base = self._makeOne()
+        with self.assertRaises(NotImplementedError):
+            api_base._lookup(None, None)
+
```

Comment:
Definitely want to move the old tests, but I wanted to wait until we actually delete `Connection` (in case we decide not to).